src/app/projects/components/data-use-restriction-group/data-use-restriction-group.component.html

```diff
@@ -0,0 +1,23 @@
+<form [formGroup]="form">
+
+  <!-- Data Use Restriction Dropdown -->
+  <app-metadata-field
+    [metadata]="metadataForm.get('project.content.data_use_restriction')"
+    [control]="metadataForm.getControl('project.content.data_use_restriction')"
+    [id]="'dataUseRestriction'">
+  </app-metadata-field>
+
+  <!-- Conditional DUOS ID Input Field -->
+  <!-- Displayed only if 'GRU' or 'GRU-NCU' is selected -->
+  <div *ngIf="form.get('data_use_restriction').value === 'GRU' || form.get('data_use_restriction').value === 'GRU-NCU'">
```

Comment:
You could read the values from the "if.properties.data_use_restriction", so avoid repetition.
Also, you could use array's `includes` function.
```suggestion
  <div *ngIf="['GRU', 'GRU-NCU'].includes(form.get('data_use_restriction').value)">
```